macos/Sources/Features/Command Palette/TerminalCommandPalette.swift

```diff
@@ -9,6 +9,9 @@ struct TerminalCommandPaletteView: View {
     /// result in the view disappearing.
     @Binding var isPresented: Bool
 
+    /// Set this to true for the jump palette mode (only focus action).
```

Comment:
I'd just change this to an enum of `mode` (two values). And it doesn't have to be a `@Binding` since its only one-way.